federation/cluster/federation-up.sh

```diff
@@ -114,10 +114,9 @@ function init() {
       --dns-zone-name="${DNS_ZONE_NAME}" \
       --dns-provider="${DNS_PROVIDER}" \
       --image="${kube_registry}/hyperkube-amd64:${kube_version}" \
-      --apiserver-arg-overrides="--storage-backend=etcd2" \
       --apiserver-enable-basic-auth=true \
       --apiserver-enable-token-auth=true \
-      --apiserver-arg-overrides="--v=4" \
+      --apiserver-arg-overrides="--runtime-config=api/all=true,--v=4" \
```

Comment:
Can you enable it explicitly in e2e test instead of enabling it here?
I assume this change will impact anyone running `kubefed init` outside e2e tests as well? (we dont want that)